files/zh-cn/web/performance/critical_rendering_path/index.md

```diff
@@ -3,6 +3,8 @@ title: 关键渲染路径
 slug: Web/Performance/Critical_rendering_path
 ---
 
+{{QuickLinksWithSubPages("Web/Performance")}}
```

Comment:
No items have been listed if only passed the slug without the prefix `/zh-CN/docs/`

A related comment: https://github.com/mdn/yari/issues/7438#issuecomment-1371880847